core/src/main/java/io/grpc/internal/OobChannel.java

```diff
@@ -54,6 +54,7 @@
   private SubchannelPicker subchannelPicker;
 
   private final LogId logId = LogId.allocate(getClass().getName());
+  private final ChannelStats channelStats = new ChannelStats();
```

Comment:
nit: This is effectively untestable since you are using the system clock.  End to end tests will be harder to implement.